dashboard/app/models/script_level.rb

```diff
@@ -37,6 +36,22 @@ def script
     Script.get_from_cache(script_id)
   end
 
+  def level
+    levels[0]
+  end
+
+  def level=(l)
+    levels[0] = l
+  end
+
+  def level_id
+    levels[0].id
+  end
+
+  def level_id=(new_level_id)
+    levels[0] = Level.find(new_level_id)
+  end
```

Comment:
It's used at least 60 times, and simply renaming it wouldn't be difficult. I was primarily trying to keep this PR from getting too large.

Ideally we won't use level (or first_level) anywhere after I'm done, which means going through those 60 uses and making more substantial changes that make the code aware of the new relation. Since I'm going to have to do that anyway, I didn't see much benefit from renaming them here.